src/components/teampage/team-members/team-member.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import {
+  ContactCardContainer,
+  CardPosition,
+  InfoContainer,
+  GraidentSquare,
+} from "./team-member-styles"
+
+import Img from "gatsby-image"
+
+import {
+  MemberName,
+  MemberPosition,
+  Overlay,
+  OverlayText,
+  MemberImg,
+  ConnectLink,
+  MemberInfo,
+} from "../team-shared-styles"
+
+export default ({
+  name,
+  photo,
+  email,
+  major,
+  position,
+  linkedinUrl,
+  biography,
+  team,
+  index,
+  teamIndex,
+  isLast,
+}) => (
+  <CardPosition index={index} isLast={isLast}>
```

Comment:
This looks like the almost same code as a the team member component. Can you please reuse it?

Reply:
Hmm, I'm not sure I understand. This is the team member component. Do you mean the advisor member component maybe? They have mostly different styling and a dif layout. Lmk!

Reply:
I think I'm hallucinating, sorry – thank you for pointing this out